bin/wp-cli.php

```diff
@@ -128,8 +165,13 @@ public function flush() {
 	 *      # Index six specific posts
 	 *      wp searchpress index 12340 12341 12342 12343 12344 12345
 	 *
+	 *      # Index posts published between 11-1-2015 and 12-30-2015 inclusive
+	 *      wp searchpress index --from=2015-11-01 --to=2015-12-30
+	 *
+	 *      # Index posts published after 11-1-2015
+	 *      wp searchpress index --from=2015-11-01
 	 *
-	 * @synopsis [--flush] [--put-mapping] [--bulk=<num>] [--limit=<num>] [--page=<num>] [<post-id>]
+	 * @synopsis [--flush] [--put-mapping] [--bulk=<num>] [--limit=<num>] [--page=<num>] [--from=<date>] [--to=<date>] [<post-id>]
```

Comment:
I wonder if "from" and "to" are the best arg names. Those words aren't date-oriented, they're range-oriented, and could imply dates, IDs, etc. What do you think about making them `--after-date` and `--before-date` instead? This would give us the ability to later do ID ranges without causing confusion.